### ISSUE TITLE
`URL` should not be subclass of `Text`

### ISSUE THREAD

**Issue by VladimirAlexiev** [schemaorg/schemaorg#2850]
IMHO this should be removed:
```ttl
schema:URL rdfs:subClassOf schema:Text
```

OWL has a dichotomy between ObjectProperties and DatatypeProperties. 
Schema doesn't have such dichotomy, but in a sense the Schema JSONLD context does:
it declares 71 props to be URLs (`@id`), so they can't carry literal values.

I checked in the ontology, and none of them allow datatype values (#1747): This doesn't find any amongst those 71 props, if no reasoning is employed:
```sparql
  ?p schema:rangeIncludes/rdf:type schema:DataType
```

Many of these props have range `URL`. So if RDFS reasoning is employed, the above subclass declaration will imply that they allow datatype values.

Many OWL ontologies use xsd:AnyURI literal for "free-floating URLs" vs a resource (owl:Thing) for URLs that carry some statements. But there's no such distinction in Schema.

In short: if you even consider `URL` to be a literal, it's a very special kind of literal, so please don't declare it subclass of `Text`.
